### PR TITLE
feat: material table

### PR DESCRIPTION
**Pull request by manuel-calavera**

### CODE REVIEW

File: src/v2/components/Transactions/index.jsx [hunk 1]
```diff
@@ -16,33 +14,12 @@ import useStyles from './styles';
...
-    next,
-    prev,
-  } = TransactionsTimelineStore;
+  const {transactions, transactionCount, start} = TransactionsTimelineStore;
 
   if (start !== match.params.start) {
     TransactionsTimelineStore.init({start: match.params.start});
   }
 
-  const nav = (
```
Comment (sunnygleason) [inline src/v2/components/Transactions/index.jsx]: can we agree, please don't remove nav until we have a replacement? It's useful for debugging pagination

File: src/v2/components/Blocks/index.jsx [hunk 2]
```diff
@@ -16,27 +14,12 @@ import useStyles from './styles';
...
 const BlocksPage = ({match}: {match: Match}) => {
   const classes = useStyles();
-  const {blocks, blockCount, start, next, prev} = BlocksTimelineStore;
+  const {blocks, blockCount, start} = BlocksTimelineStore;
 
   if (start !== match.params.start) {
     BlocksTimelineStore.init({start: match.params.start});
   }
 
-  const nav = (
```
Comment (sunnygleason) [inline src/v2/components/Blocks/index.jsx]: can we agree, please don't remove nav until we have a replacement? It's useful for debugging pagination

File: src/v2/components/Transactions/index.jsx (hunk 1 above)
Comment (manuel-calavera) [reply src/v2/components/Transactions/index.jsx]: pagination is still there but provided by material table, so we don't need our own buttons

**Comment by manuel-calavera** [comment]
@sunnygleason put nav back, disabled local material-table ui nav for now.

**Pull request merged**